chalicelib/checks/audit_checks.py

```diff
@@ -1074,3 +1075,37 @@ def restrict_hela(connection, **kwargs):
         action.status = 'DONE'
     action.output = action_logs
     return action
+
+
+@check_function(days_delay=14)
+def external_submission_but_missing_dbxrefs(connection, **kwargs):
+    ''' Check if items with external_submission also have dbxrefs.
+    When exporting metadata for submission to an external repository,
+    external_submission is patched. After some time (delay), the corresponding
+    dbxref should also have been received and patched.
+    '''
+    check = CheckResult(connection, 'external_submission_but_missing_dbxrefs')
+    delay = kwargs.get('days_delay')
+    try:
+        delay = int(delay)
+    except (ValueError, TypeError):
+        delay = 14
+    date_now = datetime.datetime.now(datetime.timezone.utc)
+    days_diff = datetime.timedelta(days=delay)
+    to_date = datetime.datetime.strftime(date_now - days_diff, "%Y-%m-%d %H:%M")
+
+    query = ('search/?type=Item&dbxrefs=No+value' +
+             '&external_submission.date_exported.to=' + to_date)
+    items = ff_utils.search_metadata(query + '&field=dbxrefs&field=external_submission', key=connection.ff_keys)
```

Comment:
adding `&frame=object` here would make this a little quicker.

Reply:
Since I restricted the query to only those two fields, I thought that changing the frame would not make any difference. Anyway, I don't expect this search to return too many items, in theory.

Reply:
there's a difference but it's small.